frontend/src/entry/settings/ManualEntryAccess.tsx

```diff
@@ -0,0 +1,67 @@
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore'
+import { Accordion, AccordionDetails, AccordionSummary, Button, Stack, TextField, Typography } from '@mui/material'
+import { getUserInformation } from 'actions/user'
+import { useState } from 'react'
+import MessageAlert from 'src/MessageAlert'
+import { CollaboratorEntry, EntityKind } from 'types/types'
+import { getErrorMessage } from 'utils/fetcher'
+
+interface ManualEntryAccessProps {
+  accessList: CollaboratorEntry[]
+  setAccessList: (accessList: CollaboratorEntry[]) => void
+}
+
+export default function ManualEntryAccess({ accessList, setAccessList }: ManualEntryAccessProps) {
+  const [manualEntityName, setManualEntityName] = useState('')
+  const [errorMessage, setErrorMessage] = useState('')
+
+  const handleAddEntityManuallyOnClick = async () => {
+    setErrorMessage('')
+    if (manualEntityName !== undefined && manualEntityName !== '') {
+      if (accessList.find((collaborator) => collaborator.entity === `${EntityKind.USER}:${manualEntityName}`)) {
+        return setErrorMessage(`The requested user has already been added below.`)
+      }
+      const response = await getUserInformation(manualEntityName)
+      if (!response.ok) {
+        return setErrorMessage(await getErrorMessage(response))
+      }
+      const updatedAccessList = [...accessList]
+      const newAccess = { entity: `${EntityKind.USER}:${manualEntityName}`, roles: [] }
+      updatedAccessList.push(newAccess)
+      setAccessList(updatedAccessList)
```

Comment:
Consider condensing this like so:
```
setAccessList([...accessList, { entity: `${EntityKind.USER}:${manualEntityName}`, roles: [] }])
```

It turns 4 lines into 1 and I think it's also easier to read.